packages/synapse-react-client/src/components/RORInstitutionField/RORInstitutionField.tsx

```diff
@@ -0,0 +1,104 @@
+import { ROROrganization } from '@/ror-client/types/ROROrganization'
+import { useSearchRegistry } from '@/synapse-queries/ror'
+import { Autocomplete, Box, TextField, Typography } from '@mui/material'
+import { useDebouncedEffect } from '@react-hookz/web'
+import noop from 'lodash-es/noop'
+import { useState } from 'react'
+
+export type RORLinkedInstitutionFieldProps = {
+  value?: string
+  onChange?: (value: string | undefined, rorIdentifier?: string) => void
+}
+
+function getOrganizationDisplayName(organization: ROROrganization) {
+  return organization.names.find(name =>
+    name.types.includes('ror_display'),
+  )!
+}
+
+/**
+ * Provides a text field for entering an organization name with an autocomplete dropdown of ROR organizations, fetched from the
+ * ROR API.
+ * @param props
+ * @constructor
+ */
+export default function RORInstitutionField(
+  props: RORLinkedInstitutionFieldProps,
+) {
+  const { value: valueFromProps, onChange = noop } = props
+
+  const isControlled = valueFromProps !== undefined
```

Comment:
Are there cases where this component is being used independently, not controlled (by the parent defining the value prop)?